### PR TITLE
Update local-storage.js

### PR DESCRIPTION
**Pull request by AllenBird**
**Type: bug**

**Description:**
Delete versions and history in package.json, to fix issue #1359.

### CODE REVIEW

**Comment by AllenBird** [comment]
unit-test failed

File: src/lib/local-storage.js [hunk 1]
```diff
@@ -360,8 +360,19 @@ class LocalStorage implements IStorage {
       name,
       (data, cb) => {
         if (data._attachments[filename]) {
-          delete data._attachments[filename];
-          cb();
+          // get version form _attachments;
+          // there aways had 'version' in verdaccio publish package.
```
Comment (lirantal) [inline src/lib/local-storage.js]:
```suggestion
// there always is a 'version' in published package
```

File: src/lib/local-storage.js [hunk 2]
```diff
@@ -360,8 +360,19 @@ class LocalStorage implements IStorage {
...
+
+          // see https://github.com/verdaccio/verdaccio/issues/1359
```
Comment (lirantal) [inline src/lib/local-storage.js]: reference can be removed
```suggestion
```

**Comment by juanpicado** [comment]
@lirantal https://github.com/verdaccio/verdaccio/pull/1389 this is the right PR :-) this is closed.

**Comment by lirantal** [comment]
eeek, losing track fast :)

**Pull request closed without merging**